objstore.go

```diff
@@ -631,9 +642,10 @@ type timingReadCloser struct {
 	failed            *prometheus.CounterVec
 	isFailureExpected IsOpFailureExpectedFunc
 	fetchedBytes      *prometheus.CounterVec
+	uploadedBytes     *prometheus.CounterVec
```

Comment:
I don't think we increment it anywhere? Did you miss incrementing it?

We also really need this metric for Upload operation. No other operation writes significant number of bytes unless I am wrong (:

Reply:
I missed it too, my bad.
I have added it now.